site/src/examples/dialog/Subtitle.tsx

```diff
@@ -0,0 +1,52 @@
+import { ReactElement, useState } from "react";
+import { Button, FormField, FormFieldLabel, Input, useId } from "@salt-ds/core";
+import {
+  Dialog,
+  DialogTitle,
+  DialogActions,
+  DialogContent,
+  DialogCloseButton,
+} from "@salt-ds/lab";
+
+export const Subtitle = (): ReactElement => {
+  const [open, setOpen] = useState(false);
+  const id = useId();
+
+  const handleRequestOpen = () => {
+    setOpen(true);
+  };
+
+  const onOpenChange = (value: boolean) => {
+    setOpen(value);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <>
+      <Button onClick={handleRequestOpen}>Open dialog with subtitle</Button>
+      <Dialog open={open} onOpenChange={onOpenChange} size="small">
+        <DialogTitle
+          title="Subscribe"
+          subtitle="Recieve emails about the latest updates"
+        />
```

Comment:
Adding an aria-labelled by like in the other examples doesn't work here. The screen reader reads the subtitle first followed by the title.
Id needs to be passed to Dialog Title and then onto the H2 tag

Reply:
Yes, this is one of the advantage when choosing composition, i.e. user can have access to individual element. 

`id` should still be mapped to root element by convention, if needed, can add old approach of `TitleProps` and spread that to sub-components

Reply:
i've passed added `id` as a prop to DialogTitle and then onto the H2 tag